bench/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "0.0.0",
+  "main": "index.js",
```

Comment:
Let's omit this (as we do in the top-level __package.json__) as it's the default value.